wire-ios-sync-engine/Source/UserSession/ZMUserSession+Debugging.swift

```diff
@@ -66,7 +66,7 @@ extension ZMUserSession {
 
     fileprivate var debugStateUserDefaultsKey: String? {
         guard
-            let identifier = (self.selfUser as! ZMUser).remoteIdentifier
+            let identifier = (self.providedSelfUser as! ZMUser).remoteIdentifier
```

Comment:
```suggestion
            let identifier = (self.providedSelfUser as? ZMUser)?.remoteIdentifier
```

Reply:
Once we get rid of `ZMUserSession.shared()`, we'll also get rid of `ZMUser.selfUser()` and `SelfUserProvider`.